### PR TITLE
workflows: Enable debug mode in conformance tests

### PR DESCRIPTION
**Pull request by pchaigno**

### CODE REVIEW

File: .github/workflows/conformance-ingress.yaml [hunk 1]
```diff
@@ -111,7 +111,8 @@ jobs:
             --set operator.image.pullPolicy=IfNotPresent \
             --set operator.image.useDigest=false \
             --set securityContext.privileged=true \
-            --set ingressController.enabled=true
+            --set ingressController.enabled=true \
+            --set debug.enabled=true
```
Comment (sayboras) [inline .github/workflows/conformance-ingress.yaml:115]: this is good idea, I have incorporated the same in https://github.com/cilium/cilium/pull/19825

Comment (pchaigno) [reply .github/workflows/conformance-ingress.yaml:115]: If you're going to carry half of this patch in your PR, would you mind carrying the other half as well (i.e., changes to `conformance-kind.yaml`)? :grin:

Comment (sayboras) [reply .github/workflows/conformance-ingress.yaml:115]: Sure 💯

Comment (sayboras) [reply .github/workflows/conformance-ingress.yaml:115]: Done :heavy_check_mark:

**Comment by pchaigno** [comment]
Closing in favor of https://github.com/cilium/cilium/pull/19825.

**Pull request closed without merging**